<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorLocation</class>
 <widget class="QWidget" name="EditorLocation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="addressTypeLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Show address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="addressSelectionCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addAddressButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Add new</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="streetLabel">
       <property name="text">
        <string>Street</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="streetLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="postOfficeBoxLabel">
       <property name="text">
        <string>Post Office Box</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="postOfficeBoxLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="localityLabel">
       <property name="text">
        <string>Locality</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="localityLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="regionLabel">
       <property name="text">
        <string>Region</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="regionLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="postalCodeLabel">
       <property name="text">
        <string>Postal Code</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="postalCodeLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="countryLabel">
       <property name="text">
        <string>Country</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="countryLineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="editLabelLabel">
       <property name="text">
        <string>Edit Label</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="editLabelLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="deleteAddressButton">
       <property name="text">
        <string>Delete this address</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
